lib/ohai/system.rb

```diff
@@ -176,9 +176,12 @@ def attributes_print(a)
     def configure_ohai
       Ohai.config.merge!(@config)
 
-      if Ohai.config[:directory] &&
-          !Ohai.config[:plugin_path].include?(Ohai.config[:directory])
-        Ohai.config[:plugin_path] << Ohai.config[:directory]
+      # add any additional CLI passed directories to the plugin path excluding duplicates
+      unless Ohai.config[:directory].nil?
+        Ohai.config[:directory].each do |dir|
```

Comment:
This should probably be `Array(Ohai.config[:directory]).each` so it still supports scalars, otherwise you break people who might have been setting this value in config.